internal/server/data/migrationdata/202206151027-sqlite.sql

```diff
@@ -0,0 +1,46 @@
+PRAGMA foreign_keys=OFF;

```

Comment:
We should delete `internal/server/data/migrationdata/202206151027.sql` if we're not going to use it.

Reply:
Good call. I wasn't looking for an existing test case. Will update accordingly